### PR TITLE
Poc ee test

### PR DESCRIPTION
**Pull request by ldimaggi**
Cleaned up test - minimal test at this point - will expand each day.

### CODE REVIEW

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: a8b9f441ef50eb77639abc0cc8a4c0beb1744027
  * Your subject line is longer than 50 characters

 * Commit: fd1d50cb718534260cc66b390aab13041f2f5512
  * Your subject line is longer than 50 characters

 * Commit: 8c50ced033e1de117bddecf1f0bc5bbf4fe98d36
  * Your subject line is longer than 50 characters

Guidelines are available at https://github.com/fabric8io/fabric8-ui

---
This message was auto-generated by https://gitcop.com

**Comment by fabric8cd** [comment]
@ldimaggi snapshot fabric8-ui is deployed and available for testing at https://fabric8-ui-fabric8-ui-pr-888-fabric8-ui.int.rdu2c.fabric8.io

File: package.json [hunk 1]
```diff
@@ -125,6 +125,7 @@
     "ngx-modal": "0.0.29",
     "ngx-widgets": "0.14.0",
     "offline-plugin": "4.6.2",
+    "protractor-jasmine2-screenshot-reporter": "0.3.5",
```
Comment (joshuawilson) [inline package.json]: this should be in the next section for devDependencies as it is not used in the running app.

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: a8b9f441ef50eb77639abc0cc8a4c0beb1744027
  * Your subject line is longer than 50 characters

 * Commit: fd1d50cb718534260cc66b390aab13041f2f5512
  * Your subject line is longer than 50 characters

 * Commit: 8c50ced033e1de117bddecf1f0bc5bbf4fe98d36
  * Your subject line is longer than 50 characters

 * Commit: 1589187a869fdd2fb9673570f64f3fccf79483b9
  * Your subject line is longer than 50 characters
  * Invalid type. Valid types are **feat, fix, docs, style, refactor, perf, test, chore, revert**

Guidelines are available at https://github.com/fabric8io/fabric8-ui

---
This message was auto-generated by https://gitcop.com

**Comment by fabric8cd** [comment]
@ldimaggi snapshot fabric8-ui is deployed and available for testing at https://fabric8-ui-fabric8-ui-pr-888-fabric8-ui.int.rdu2c.fabric8.io

**Pull request merged**